JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -515,384 +651,534 @@
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/FoodIndustry">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Industry"/>
         <rdfs:comment>An industry established to produce food.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
         <rdfs:label>Food Industry</rdfs:label>
     </owl:Class>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#FoodPlant -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/FoodPlant -->
 
-    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany#FoodPlant">
-        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany#Factory"/>
-        <rdfs:subClassOf>
-            <owl:Restriction>
-                <owl:onProperty rdf:resource="http://www.theworldavatar.com/kg/ontocompany#belongsToIndustry"/>
-                <owl:someValuesFrom rdf:resource="http://www.theworldavatar.com/kg/ontocompany#FoodIndustry"/>
-            </owl:Restriction>
-        </rdfs:subClassOf>
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/FoodPlant">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Factory"/>
         <rdfs:comment>A factory established to produce food.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
         <rdfs:label>Food Plant</rdfs:label>
     </owl:Class>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#GeneratedHeat -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/GeneratedHeat -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/GeneratedHeat">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/IndustrialFacilityProperty"/>
+        <rdfs:comment>Calculated value of annual heat emission for a company&apos;s main emitting sources.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Heat Emissions</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/GeneratedHeatPerPrinter -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/GeneratedHeatPerPrinter">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/FactoryProperty"/>
+        <rdfs:comment>The average amount of heat emitted by each printer within a printing facility.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Generated Heat Per Printer</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/HeatingLoad -->
+
+
```

Comment:
Why do HeatingLoad and ITLoad not have any other properties (i.e. rdfs:comment, rdfs:isDefinedBy, rdfs:label )?

Reply:
I think some extraneous classes such as HeatingLoad were included in the ontology by accident. I have removed them.